third_party/zip/zip.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2010 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package zip contains a fork of archive/zip reader.
```

Comment:
nit: Is fork the right term here since it seems like most (all?) of the code here is new code to deal with executable jars? It initially made me think most of this code was copy/pasted from archive/zip which appears to not be the case.

Reply:
All the code here is copy/pasted from various parts of the archive/zip. Let me update the comment